### ISSUE TITLE
mean_pairwise_squared_error

### ISSUE THREAD

**Issue by JachinZhao** [tensorflow/tensorflow#8424]
when i use the tf.losses.mean_pairwise_squared_error(labels, predictions, weights=1.0, scope=None, loss_collection=tf.GraphKeys.LOSSES) function, i am sure the data is right. but the loss on the tensorboard is always zero. I try hard but can know why?
score_a=tf.reshape(score,[-1])#shape: [1,39]
ys_a=tf.reshape(ys,[-1])#shape: [1,39]
with tf.name_scope('loss'):
	loss=tf.losses.mean_pairwise_squared_error(score_a,ys_a)

**Comment by MicaelCarvalho** [comment]
If this isn't a bug, it will be better/faster answered on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) — the correct place to ask for help, with a bigger community. If this is a bug, please follow guidelines for reporting problems. When you click on "New issue" there is a template to be filled.

On the topic: You have to call [tf.summary.scalar](https://www.tensorflow.org/api_docs/python/tf/summary/scalar) to send/print your loss to TensorBoard. Please take a look at the [docs of TensorBoard](https://www.tensorflow.org/get_started/summaries_and_tensorboard) for more information.

**Comment by concretevitamin** [comment]
I agree with the assessment of @MicaelCarvalho - the usage questions are best served via StackOverflow.
